app/views/stacks/_header.html.erb

```diff
@@ -1,16 +1,28 @@
-<header class="header repo-header">
-  <div class="wrapper">
-    <div class="repo-name" data-repo-full-name="<%= stack.github_repo_name %>">
-      <p><%= stack.repo_owner %></p>
-      <h1><a href="<%= stack_path(@stack) %>"><%= stack.repo_name %> <small><%= stack.environment %></small></a></h1>
-      <% if stack.deploy_url.present? %>
-        <%= link_to stack.deploy_url, stack.deploy_url, :target => '_blank', :class => 'deploy-url' %>
+<% content_for :page_title do %>
+  <p><%= stack.repo_owner %></p>
+  <h1><a href="<%= stack_path(@stack) %>"><%= stack.repo_name %> <small><%= stack.environment %></small></a></h1>
+  <% if stack.deploy_url.present? %>
+    <%= link_to stack.deploy_url, stack.deploy_url, :target => '_blank', :class => 'deploy-url' %>
+  <% end %>
+<% end %>
+
+
+<% content_for :primary_navigation do %>
+  <%= link_to 'Refresh', stack_refresh_path(@stack), method: 'post', data: {tooltip: 'Refresh statuses and commits'}, class: "btn secondary" %>
```

Comment:
This tooltip needs to go below the button: 

![screen shot 2015-05-21 at 5 45 56 pm](https://cloud.githubusercontent.com/assets/403774/7759654/505c40a0-ffe1-11e4-9099-5f995aecd362.png)